test/events.js

```diff
@@ -27,6 +27,37 @@ export function testEvents(options, innerElement, getInnerPoint) {
   describe('events', function() {
     const pluginOpts = chartConfig.options.plugins.annotation;
 
+    it(`should detect only leave event on ${descr}`, function(done) {
```

Comment:
I think something like:

```suggestion
    it(`should not call removed hook on ${descr}`, function(done) {
```

would better describe the test.